### ISSUE TITLE
Frizerski salon_html_css

### ISSUE THREAD

**Issue by CatYo** [CatYo/frizerski_salon#1]
<!DOCTYPE html>
<html lang="en">
<head>
       <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

```
    <title>Frizerski salon</title>


    <link href="css/bootstrap.min.css" rel="stylesheet">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
    <script src="js/bootstrap.min.js"></script>
```

<style>
    div h1 {
    font-size:50px;
    align:center;
        font-family: monospace;
    
        text-color: cadetblue;
    }
    
    
    .content{
    background-color:azure;
    margin-top: 150px;
    }
    
    .content p {
    height:auto;
        
        font-size:15px;
    color:cadetblue;
        float:left;
        font-family: cursive;
    }
    
    div img{
  
        width:300px;
        height:450px;
        float:left;
        padding-left: 10px;
    }

    .row1{
    float:left;
    align:auto;
    font-size:15px;
    padding-left:800px;
        font-family: monospace;
   
    }
    
    div ul{
    list-style: none;
        padding-left:30px;
      padding-bottom: 5px;
        font-family: monospace;
        float:right;
        font-size: 15px;
        
       
    }
 
    div ul li{
    float:left;
    padding-right:20px;
    }
    
    
     img.responsive {
        width:100%;
            height:auto;
        }
    
    .rowl p{
    clear:both; 
    align:justify;
    font-family: cursive;
        padding-leftt:50px;
    }
    
    .img12345{
    align-content: center;
        
    }
    
</style>                     

</head>

<body>

<div class="content">
<div class="row">
   
    <div role="navigator">
        <div class="nav-pills">
         <ul class="nav nav-pills">
          <li role="presentation" class="active"><a href="#">Menu</a></li>
          <li role="presentation"><a href="#">Galerija</a></li>
          <li role="presentation"><a href="#">Reference</a></li>
           <li role="presentation"><a href="#">Kontakt</a></li>
        </ul>
        </div>
    </div>
    <div class="col-xs-6">
        <h1>FRIZERSKI SALON NIKA</h1>
            <p>V modernem, prostornem frizerskem salonu nudimo svojim strankam vrhunske storitve na         področju standardnih frizerskih storitev, svetovanja, oblikovanja svečanih pričesk, podaljševanja las, kot tudi prodaje kvalitetne lasne kozmetike. Tudi pri svojem delu naši frizerji uporabljajo lastne kvalitetne proizvode znamke Mmm , s katerimi pripomorejo k izboljšanju strankinega  dobrega izgleda in počutja. Za vas bomo najlažje poskrbeli, če se boste predhodno naročili na telefonsko št. 111111111. Z vstopom v kletno etažo centra Mitja vstopite v svet tišine, globoke sprostitve ter umiritve duha in telesa. Tu vam ponujamo široko ponudbo wellness storitev, od lepotne nege obraza in telesa,  klasične in tajske masaže, pedikure ter sproščanja v eni izmed izbranih savn.</p>
           <div class="img12345"> <img src="http://www.globalfashionreport.com/inline/fashion/sr/4bc0482e7f24c8a965d11b9791e028bf.jpg" class="img-responsive" />
            <img src="https://s-media-cache-ak0.pinimg.com/736x/97/7d/11/977d11286fc5dd7317ce9bcd06788354.jpg" class="img-responsive" />
            <img src="https://s-media-cache-ak0.pinimg.com/236x/0e/58/bd/0e58bdbf28a70d3ca554afdb4a8aad0f.jpg" class="img-responsive" />
            <img src="https://s-media-cache-ak0.pinimg.com/736x/b3/5a/19/b35a19caef39ee242bac3f1c2c8c7ff0.jpg" class="img-responsive" />
              <img src="https://s-media-cache-ak0.pinimg.com/236x/21/87/f6/2187f6035ddc97fa541d78093742ab3c.jpg" class="img-responsive" />
              <img src="https://s-media-cache-ak0.pinimg.com/736x/56/16/79/56167980a30fa0d3bb098e7eeafdd411.jpg" class="img-responsive" />
            
               
                </div>
    <div class="row1">
        <div class="col-xs-6" class="kontakt">KatjaB Copyright 2015</div>
     
    </div>
</div>

</div>
</body>
</html>
